src/interval_sets.jl

```diff
@@ -427,16 +434,17 @@ Base.intersect(x::IntervalSet, y::IntervalSet) = mergesets((inx, iny) -> inx &&
 Base.union(x::IntervalSet, y::IntervalSet) = mergesets((inx, iny) -> inx || iny, x, y)
 Base.setdiff(x::IntervalSet, y::IntervalSet) = mergesets((inx, iny) -> inx && !iny, x, y)
 Base.symdiff(x::IntervalSet, y::IntervalSet) = mergesets((inx, iny) -> inx ⊻ iny, x, y)
+Base.issubset(x::T, y::IntervalSet{<:AbstractInterval{T}}) where {T} = any(i -> x in i, y.items)
```

Comment:
Doesn't this mean that if x doesn't have the same concrete type as the element type of y it will throw a method error? I think `issubset(2, [1.0..3.0, 4.0..5.0])` should be `true` not a method error.

Reply:
Done. One thing I also noticed is that we aren't testing things like `issubset(interval, intervals)` and `issubset(intervals, interval)` which would fallback to calling the base `setdiff` and erroring. I'll go through and add more tests for the other set operations in case there are other incorrect fallback conditions.